exercises/practice/twelve-days/twelve_days.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+namespace twelve_days {}  // namespace twelve_days
```

Comment:
This is how we do this in other exercises:
```suggestion
namespace twelve_days {

// TODO: add your solution here

}  // namespace twelve_days
```